openmls/src/group/core_group/kat_passive_client.rs

```diff
@@ -155,10 +155,10 @@ pub fn run_test_vector(test_vector: PassiveClientWelcomeTestVector) {
     let ratchet_tree: Option<RatchetTree> = test_vector
         .ratchet_tree
         .as_ref()
-        .map(|bytes| RatchetTree::tls_deserialize(&mut bytes.0.as_slice()).unwrap());
+        .map(|bytes| RatchetTree::tls_deserialize_exact(&mut bytes.0.as_slice()).unwrap());
```

Comment:
clippy was unhappy about this.

```suggestion
        .map(|bytes| RatchetTree::tls_deserialize_exact(bytes.0.as_slice()).unwrap());
```